components/input/ResizableTextArea.tsx

```diff
@@ -103,7 +115,7 @@ class ResizableTextArea extends React.Component<TextAreaProps, TextAreaState> {
       ...(resizing ? { overflow: 'hidden' } : null),
     };
     return (
-      <ResizeObserver onResize={this.resizeOnNextFrame} disabled={!(autoSize || autosize)}>
```

Comment:
No, it still works, I didn't make any changes to that feature.

Reply:
No, it still works, I didn't make any changes to that feature.

Reply:
Sorry, I miss your handleResize  change.

Reply:
Why remove `disabled`?

Reply:
If it's true, it prevent onResize not trigger

Reply:
See `handleResize`

Reply:
Should add `onResize` check here since listen for resize is cost. Should not listen every when `onResize` or other condition not match.

Reply:
Even if autoSize/autosize = {false}, ResizeObserver can still resize, it just do not trigger onResize any more.

Reply:
`onResize` should not work when `autoSize` is `false`. If a user use plenty of TextArea. It will cause performance issue even user not use `onResize` since we observe it anyway.

Reply:
@zombieJ Let's discuss on DingTalk.